net-misc/eventd/eventd-0.21.0.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit linux-info systemd
+
+DESCRIPTION="A small daemon to act on remote or local events"
+HOMEPAGE="https://www.eventd.org/"
+SRC_URI="https://www.eventd.org/download/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3+ LGPL-3+ MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="debug fbcon +introspection ipv6 libcanberra libnotify +notification
+	purple sound speech ssdp systemd test websocket +X zeroconf"
+
+REQUIRED_USE="
+	X? ( notification )
+	fbcon? ( notification )
+	notification? ( || ( X fbcon ) )
+"
+
+COMMON_DEPEND="
+	dev-libs/glib:2
+	sys-apps/util-linux
+	introspection? ( dev-libs/gobject-introspection )
+	libcanberra? ( media-libs/libcanberra )
+	libnotify? ( x11-libs/gdk-pixbuf:2 )
+	notification? (
+		x11-libs/cairo
+		x11-libs/pango
+		x11-libs/gdk-pixbuf:2
+		X? (
+			x11-libs/cairo[xcb]
+			x11-libs/libxcb:=
+			x11-libs/xcb-util
+			x11-libs/xcb-util-wm
+		)
+	)
+	purple? ( net-im/pidgin )
+	sound? (
+		media-libs/libsndfile
+		media-sound/pulseaudio
+	)
+	speech? ( app-accessibility/speech-dispatcher )
+	ssdp? ( net-libs/gssdp:= )
```

Comment:
gssdp is about DLNA/UPnP. The right USE flag for it is either upnp or upnp-av in Gentoo:
```
upnp - Enable UPnP port mapping support
upnp-av - Enable UPnP audio/video streaming support
```